<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FavoritesPage</class>
 <widget class="QWidget" name="FavoritesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 36pt &quot;Arial&quot;;
font: 36pt &quot;Hei&quot;;
font: 36pt &quot;Heiti SC&quot;;</string>
     </property>
     <property name="text">
      <string>Favorites</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="tracksButton">
       <property name="text">
        <string>Tracks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="albumsButton">
       <property name="text">
        <string>Albums</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="artistsButton">
       <property name="text">
        <string>Artists</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>668</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
